distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Base_Generator.enso

```diff
@@ -117,7 +118,12 @@ type SQL_Generator
         From_Spec.Table name as_name _ ->
             dialect.wrap_identifier name ++ alias dialect as_name
         From_Spec.Query raw_sql as_name ->
-            SQL_Builder.code raw_sql . paren ++ alias dialect as_name
+            subquery = case raw_sql of
+                prepared_statement : SQL_Statement ->
+                    SQL_Builder.from_fragments prepared_statement.fragments
+                raw_code : Text ->
+                    SQL_Builder.code raw_code
```

Comment:
Could we do this as conversion on SQL_Builder?

Reply:
I think I'd prefer this to be explicit.